target/linux/bmips/bcm6368/base-files/etc/board.d/02_network

```diff
@@ -5,6 +5,10 @@
 board_config_update
 
 case "$(board_name)" in
+actiontec,r1000h)
+	ucidef_set_bridge_device switch
+	ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 coax" "wan"
```

Comment:
Does it make sense to add `coax` here?
This should probably be added to a separate interface or not added at all if we can't get it working on OpenWrt...

Reply:
Something like https://github.com/openwrt/openwrt/blob/main/target/linux/bmips/bcm63268/base-files/etc/board.d/02_network#L16-L17

Reply:
ok, I''ll make this change